.github/ISSUE_TEMPLATE.md

```diff
@@ -23,8 +23,8 @@ A brief description of what failed or what could be improved.
 
 # Information to attach
 
- - [ ] any relevant kernel output (dmesg)
- - [ ] container log (lxc info NAME --show-log)
- - [ ] main daemon log (/var/log/lxd.log)
+ - [ ] any relevant kernel output (`dmesg`)
+ - [ ] container log (`lxc info NAME --show-log`)
+ - [ ] main daemon log (`less /var/log/lxd/lxd.log`)
```

Comment:
Probably should be "cat" rather than "less" as the user will want the whole thing in one shot.